### PR TITLE
Remove mentions to RE2

### PR DESCRIPTION
**Pull request by pedrosousa**

### CODE REVIEW

**Review by helloandre — APPROVED** [review]

File: products/firewall/src/content/known-issues-and-faq/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The following image illustrates how double quotes are automatically escaped to `
 
 ### Why isn't my regular expression pattern matching working?
 
-If you are using a regular expression, we recommend you test it against an _RE2_ testing tool like [Regular Expressions 101](https://regex101.com).
+If you are using a regular expression, we recommend you test it against a testing tool like [Regular Expressions 101](https://regex101.com).
```
Comment (migueldemoura) [inline products/firewall/src/content/known-issues-and-faq/index.md]: This website doesn't have Rust's regex engine as an option and the default is PCRE which is very different from what we support. I think the closest they have to ours is Go's (RE2).

Comment (pedrosousa) [reply products/firewall/src/content/known-issues-and-faq/index.md]: Should we recommend using Golang's flavor, or leave it generic?

Comment (pedrosousa) [reply products/firewall/src/content/known-issues-and-faq/index.md]: Kept Regular Expressions 101 and added Rustexp.

**Review by abracchi-tw — APPROVED** [review]

File: products/firewall/src/content/known-issues-and-faq/index.md (hunk 1 above)
Comment (migueldemoura) [reply products/firewall/src/content/known-issues-and-faq/index.md]: I think that's the best option here 👍

**Review by migueldemoura — APPROVED** [review]

File: products/firewall/src/content/known-issues-and-faq/index.md (hunk 1 above)
Comment (pedrosousa) [reply products/firewall/src/content/known-issues-and-faq/index.md]: Added URL query string parameter to https://regex101.com to set the flavor to "Golang".

**Review by ytbigvinny27 — COMMENTED** [review]
~

**Review by ytbigvinny27 — COMMENTED** [review]
~

**Review by ytbigvinny27 — COMMENTED** [review]
~

**Pull request merged**